### ISSUE TITLE
LC0005 variables on CalcFormula

### ISSUE THREAD

**Issue by Arthurvdv** [StefanMaron/BusinessCentral.LinterCop#475]
The LC0005 should also check on variables in a CalcFormula.

```AL
table 50102 MyTable
{
    fields
    {
        field(1; "MyField"; Integer) { }
        field(2; "Customer No."; Code[20])
        {
            FieldClass = FlowField;
            CalcFormula = lookup(customer."No."); // Should raise the rule here on customer without a capital C
        }
    }
}
```
